gradle/ext.gradle

```diff
@@ -10,7 +29,7 @@ gradle.ext {
     serialization_version = '1.2.1'
     kydra_log_version = '1.1.5'
     android_ble_scanner_version = '1.5.0'
-    library_version = '0.2.1'
+    library_version = libraryVersionLocalProperties ?: "0.2.2${System.properties.kaluga_branch_postfix}"
```

Comment:
a minor optional thing: could we have a separate variable for "0.2.2" prefix (e.g. `library_version_base`)? I think it'd be less error prone when editing

Reply:
https://github.com/splendo/kaluga/pull/373/files#diff-12d0b2443ec18fcaeda2652d9d55131c855c8b9e4f121fa9c66dc803d4e2ea8cR33